src/naksu/constants/constants.go

```diff
@@ -117,3 +121,10 @@ func GetAvailableSelectionID(configValue string, choices []AvailableSelection, v
 
 	return valueIfNotFound
 }
+
+// EnvironmentStatusType is used by the UI to store the status of the system environment
+type EnvironmentStatusType struct {
```

Comment:
`Type` lienee turha ja jopa hämäävä suffiksi, vai palveleeko se jotain tarkoitusta? Vastaavia ei ole käytetty Naksussa, eikä taida olla Go:n tyylin mukaista muutenkaan.

Kun ylempänä luin tän tyypin nimen ekan kerran, niin eka ajatus oli, että näitä EnvironmentStatus-asioita on useaa eri tyyppiä, ja tämä `EnvironmentStatusType` on enum, jossa nämä tyypit on määritelty.